docs/transparent-validation/rte/v1.1.0/specification.md

```diff
@@ -166,6 +166,48 @@ connector and can be enabled/disabled on demand:
 | 0 - low                 | disabled (by default)         |
 | 1 - high                | enabled                       |
 
+Example flashing procedure:
```

Comment:
In my opinion, this section should be placed in a separate section (as indicated in the task). Maybe we should add a section named `SPI header configuration`

Reply:
Done. Move to FAQ